build/kubeyaml/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM six8/pyinstaller-alpine:latest as pyinstaller
+ARG VERSION=0.7.0
+
+WORKDIR /tmp
+RUN wget https://github.com/squaremo/kubeyaml/archive/${VERSION}.tar.gz \
+  && tar xvf ${VERSION}.tar.gz  \
+  && mkdir -p /home/kubeyaml \
+  && cp /tmp/kubeyaml-${VERSION}/kubeyaml.py /home/kubeyaml/ \
+  && cp /tmp/kubeyaml-${VERSION}/requirements.txt /home/kubeyaml/
+
+WORKDIR /home/kubeyaml
+
+#COPY kubeyaml.py requirements.txt ./
```

Comment:
If the commented lines aren't required we can safely remove them.

Reply:
removed from dockerfile